neptune/new/internal/init_impl.py

```diff
@@ -110,6 +110,7 @@ def init(project: Optional[str] = None,
          monitoring_namespace: Optional[str] = None,
          flush_period: float = 5,
          proxies: Optional[dict] = None,
+         capture_traceback: bool = True,
```

Comment:
@Herudaio is it ok to add this parameter?

Reply:
I'm always against by default adding new init params :D However, if it's needed this one makes sense. I would put them right after `capture_stderr` so that they are at least grouped.

Reply:
But it would break backward compatibility if someone is using positional arguments instead of named.

Reply:
Right, I withdraw that comment :)